package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@dynamods/notifications-center",
-  "version": "0.0.13",
+  "version": "0.0.14",
```

Comment:
There is going to be a conflict about this one, I guess @filipeotero saw your PR and jumped ahead for 0.0.15, since none of these versions were published, so it does not matter which one you stick to in your PR. But we will need to resolve conflicts before merging